DistributedSampleHTTP/Sources/DistributedSampleHTTP/HTTPActorSystem+Server.swift

```diff
@@ -106,16 +113,32 @@ private final class HTTPHandler: @unchecked Sendable, ChannelInboundHandler, Rem
       let responseHead: HTTPResponseHead
       let responseBody: ByteBuffer
       switch result {
-        case .failure(let error):
-          responseHead = HTTPResponseHead(version: .init(major: 1, minor: 1),
-            status: .internalServerError,
-            headers: headers)
-          responseBody = ByteBuffer(string: "Error: \(error)")
-        case .success(let data):
-          responseHead = HTTPResponseHead(version: .init(major: 1, minor: 1),
-            status: .ok,
-            headers: headers)
-          responseBody = ByteBuffer(data: data)
+      case .failure(let error):
+        let status: HTTPResponseStatus
+        let errorString: String
+        if let error = error as? HTTPActorSystemError {
```

Comment:
Perhaps we can use `switch` instead of `if-else`?

Reply:
Hmm, not sure I like switch for this kind of thing :~ Mind if I keep it? :)

Reply:
Sure! :)